### ISSUE TITLE
Paste Layer/Group Option Not Available If No Empty Space at the Bottom in Layers Panel

### ISSUE THREAD

**Issue by ar-siddiqui** [qgis/QGIS#41714]
**Describe the bug**
After you have selected a layer or a group, you expect to see an option to paste layer/group. Currently this option is only visible if you click at the empty space at the bottom in the layers panel. If you don't have the empty space at bottom in your layers panel you have to make space to be able to access this option, which is not user friendly.

**Demonstration**

https://user-images.githubusercontent.com/53625184/108613651-ea329d00-73c1-11eb-9f6a-30740ca2d569.mp4

**Suggestion**
Paste Layer/Group option menu should be available when you click on these empty spaces as well not just the bottom empty space.
![image](https://user-images.githubusercontent.com/53625184/108613698-44cbf900-73c2-11eb-9cea-802b817cdb15.png)

**Comment by uclaros** [comment]
> If you don't have the empty space at bottom in your layers panel you have to make space to be able to access this option, which is not user friendly.


I agree, though you can also perform the paste action from the menus: Layer > Paste Layer/Group
It would be easier to add the *Paste Layer/Group* entry to the normal layer context menu under *copy layer* when needed.

**Comment by Pedro-Murteira** [comment]
@ar-siddiqui This situation is still present on versions 3.16.14 and 3.22.1,although it can be considered a feature request instead of a bug, since there's is an alternative way to complete this process.
